crates/iroha_test_network/src/fslock_ports.rs

```diff
@@ -75,7 +75,7 @@ impl LockContent {
 
 /// Releases the port on [`Drop`].
 #[derive(Debug, Deref, Display)]
-pub struct AllocatedPort(u16);
+pub struct AllocatedPort(pub u16);
```

Comment:
It is intentionally not public and is accessible via `Deref`
```suggestion
pub struct AllocatedPort(u16);
```